### PR TITLE
Improved Docs

### PR DESCRIPTION
**Pull request by 5HAD0W-P1R4T3**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,30 +30,30 @@ The documentation is divided into several sections with a different tone and pur
...
 ### Make the change
 
-1. Follow the ["Running locally"](#running-locally) instructions
-1. Save the files and check in the browser
-  1. Changes to React components in `src` will hot-reload
-  1. Changes to markdown files in `content` will hot-reload
-  1. If working with plugins, you may need to remove the `.cache` directory and restart the server
+1. Follow the ["Running locally"](#running-locally) instructions.
+2. Save the files and check in the browser.
+  2.1. Changes to React components in `src` will hot-reload.
```
Comment (eps1lon) [inline README.md:41]: An unordered list might make more sense here.

**Review by eps1lon — CHANGES_REQUESTED** [review]
Having a single number means you don't have to change subsequent numbers if you insert an item.

The output is still the same: https://github.com/reactjs/reactjs.org/pull/3299/files?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8

(apart from the formatting that is now broken).

**Comment by 5HAD0W-P1R4T3** [comment]
An ordered list tells the user it is done is it is in unordered list then the user may be confused to see only 1's and 1's

**Comment by gaearon** [comment]
>An ordered list tells the user it is done if it is in unordered list then the user may be confused to see only 1's and 1's

If you [_read_](https://github.com/reactjs/reactjs.org) the README, you'll see that it's rendered as an ordered list.

**Pull request closed without merging**